microsoft_atp/help.md

```diff
@@ -508,75 +431,29 @@ Example output:
 }
 ```
 
-#### Get Alerts Matching Key
+#### Get Alerts
 
-This trigger is used to get alerts that match a given key to its value.
+This trigger is used to return all new alerts.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|key|string|None|True|The key to look for in the alert|None|
-|value|string|None|True|The value to look for in the alert|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|frequency|integer|10|False|Poll frequency in seconds|None|10|
 
-##### Output
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|results|[]alert|True|All alerts that contain the given key and match its value|
-
-Example output:
+Example input:
 
 ```
 {
-   "results":[
```

Comment:
Looks like the example output was removed

Reply:
I *think* this was fixed on regen.